File: sdkwork-backend-sdk-typescript/src/types/msg-audio-content.ts
import type { AudioMediaResource } from './audio-media-resource';

/** IMé³é¢æ¶æ¯åå®¹ç±» */
export interface MsgAudioContent {
  /** æ¶æ¯åå®¹ */
  content?: string;
  /** é³é¢èµæº */
  resource?: AudioMediaResource;
}

File: sdkwork-backend-sdk-java/src/main/java/com/sdkwork/backend/model/MsgAudioContent.java
package com.sdkwork.backend.model;

public class MsgAudioContent {
    private String content;
    private AudioMediaResource resource;

    public String getContent() {
        return this.content;
    }
    
    public void setContent(String content) {
        this.content = content;
    }

    public AudioMediaResource getResource() {
        return this.resource;
    }
    
    public void setResource(AudioMediaResource resource) {
        this.resource = resource;
    }
}
